workflow-bot-app/src/test/java/com/symphony/bdk/workflow/BranchingIntegrationTest.java

```diff
@@ -40,8 +40,8 @@ static Stream<Arguments> executedActivities() {
   @SuppressWarnings("checkstyle:LineLength")
   static Stream<Arguments> expectedErrors() {
     return Stream.of(
-        arguments("/branching/invalid/swadl/if-starting-activity.swadl.yaml", InvalidActivityException.class,
-            "Invalid activity in the workflow if-in-starting-activity: Workflow's starting activity startingActivity must not have a conditional branching"),
+        //arguments("/branching/invalid/swadl/if-starting-activity.swadl.yaml", InvalidActivityException.class,
```

Comment:
It's better to remove this test, or change it to not fail when starting activity has if condition

Reply:
removed